src/Roslyn.Diagnostics.Analyzers/Core/RoslynDiagnosticsAnalyzersResources.resx

```diff
@@ -255,4 +255,31 @@
   <data name="FixNumberedComments" xml:space="preserve">
     <value>Fix numbered comments</value>
   </data>
+  <data name="DoNotCallGetTestAccessorDescription" xml:space="preserve">
+    <value>GetTestAccessor() is a helper method reserved for testing. Production code must not call this member.</value>
+  </data>
+  <data name="DoNotCallGetTestAccessorMessage" xml:space="preserve">
+    <value>Do not call GetTestAccessor() from production code</value>
```

Comment:
Shall we have the ending period for messages?

Reply:
I believe the period should be omitted from single-sentence messages, and only included for descriptions. For simplicity I just used the same string for each case here.

Reply:
Fine by me :)

Reply:
Side note: I would be more picky about tweaking these messages for a general-purpose analyzer. However, this analyzer is really targeted towards use inside Roslyn.sln.